src/components/Navbar/index.js

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react';
+import logo from '../../assets/logo.png';
+import menuIcon from '../../assets/menu-icon.svg';
+import closeIcon from '../../assets/close-icon.svg';
+import bodegaIcon from '../../assets/bodega-icon.svg';
+import vinosIcon from '../../assets/vinos-icon.svg';
+import galeriaIcon from '../../assets/galeria-icon.svg';
+import nosotrosIcon from '../../assets/nosotros-icon.svg';
+import contactoIcon from '../../assets/contacto-icon.svg';
+import searchIcon from '../../assets/search-icon.svg';
+import './styles.scss';
+
+const Navbar = () => {
+    const [isOpen, setClick] = useState(false);
```

Comment:
Estaría bien, cuando se actualiza el nombre del estado, actualizar también el nombre de la state dispatcher function (el `setClick`) para que estén coordinados:
```suggestion
const [isOpen, setIsOpen] = useState(false);
```